Files/Strings/en-US/Resources.resw

```diff
@@ -951,4 +951,13 @@
   <data name="SettingsPreferencesShowMultitaskingControl.Text" xml:space="preserve">
     <value>Show multitasking flyout in the navigation toolbar</value>
   </data>
+  <data name="InvalidItemDialogContent" xml:space="preserve">
+    <value>The item referenced is either invalid or inaccessible.
+Message:
+
```

Comment:
Why is there an empty line?

Reply:
Those new lines are preserved when showing the dialog.
I kept the very same message that was shown before which contained a couple of "\n"

Reply:
It is better to contain \n, instead of empty lines.

Reply:
Or, if possible, enable word wrap for dialog message.

Reply:
I'm not sure if "\n" works in resource files

Reply:
And another way without \n, is `The item referenced is either invalid or inaccessible.{0}Message:{1}{2}` and in `string.Format` change 0 and 2 to `Enviroment.NewLine`